test/battle/ability/shield_dust.c

```diff
@@ -164,3 +164,15 @@ SINGLE_BATTLE_TEST("Shield Dust blocks Sparkling Aria in singles")
         }
     }
 }
+
+SINGLE_BATTLE_TEST("Shield Dust does not prevent ability stat changes")
+{
+    GIVEN {
+        PLAYER(SPECIES_VIVILLON) { Ability(ABILITY_SHIELD_DUST); }
+        OPPONENT(SPECIES_ELDEGOSS) { Ability(ABILITY_COTTON_DOWN); }
+    } WHEN {
+        TURN { MOVE(player, MOVE_TACKLE); }
+    } SCENE {
+        NOT MESSAGE("Wobbuffet's Speed won't go lower!");
```

Comment:
```suggestion
        MESSAGE("Vivillon's Speed fell!");
```
Wrong thing being checked and wrong mon.

Reply:
why is this wrong? If you remove the fix this test will fail because the wobb message will be printed.

Reply:
oh wait you are correct